src/libYARP_cv/src/yarp/cv/Cv.h

```diff
@@ -14,6 +14,7 @@
 #include <type_traits>
 #include <opencv2/core/core.hpp>
 #include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/imgproc/types_c.h>
```

Comment:
This header seems to be unused in this header, probably we can just add it where it is needed?

Reply:
I have to check, probably you are right, it is already included by `<opencv2/imgproc/imgproc.hpp`

Reply:
It is used by `Cv-inl.h` that it is included in the end of the file. I can move it there

Reply:
Ah ok, I had missed that.